crud/select/plan.lua

```diff
@@ -48,6 +49,46 @@ local function get_index_for_condition(space_indexes, space_format, condition)
     end
 end
 
+local function extract_sharding_key_from_conditions(conditions, ddl_sharding_key, space_indexes, space_field_positions)
+    if ddl_sharding_key == nil then
+        return nil
+    end
+
+    -- If name is both valid index name and field name,
+    -- it is interpreted as index name.
+    local filled_fields = {}
+    for _, condition in ipairs(conditions) do
+        if condition.operator ~= compare_conditions.operators.EQ then
```

Comment:
How about REQ?

Reply:
In terms of `compare_conditions`, there is no such term as `REQ`:
https://github.com/tarantool/crud/blob/fbe4de01f5ae4620a2e5945f24df2fa33307c770/crud/compare/conditions.lua#L10-L16

Reply:
Okay, got it.